app/src/main/java/com/stevesoltys/backup/service/backup/BackupObserver.java

```diff
@@ -65,14 +59,6 @@ public void backupPackageCompleted(BackupSession backupSession, String packageNa
 
     @Override
     public void backupSessionCompleted(BackupSession backupSession, BackupResult backupResult) {
-        ConfigurableBackupTransport backupTransport = ConfigurableBackupTransportService.getBackupTransport();
-
-        if (!backupTransport.isActive()) {
```

Comment:
I looked briefly for a way to know that a backup is already running, but didn't find anything. Since the backup itself are performed by the BackupManager, I would expect it to not start a second one while the first one is still running.

Later, I want to add a "Backup Now" button again, so then we should definitely look into this again.